src/local-history/local-history-manager.ts

```diff
@@ -174,10 +174,6 @@ export class LocalHistoryManager {
                         return;
                     }
                 }
-                else if (activeDocumentContent === mostRecentRevisionContent) {
-                    fs.renameSync(mostRecentRevision, historyFilePath);
```

Comment:
@Anasshahidd21 I think only the `renameSync` should be removed, we want to exit when there is no update.